File: helper/src/test/java/com/loudsight/useful/helper/ClassHelperTest.java
package com.loudsight.useful.helper;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import java.util.ArrayList;
import java.util.List;

/**
 * Class to hold utilities related to casting of types
 *
 * @author munyengm
 */
public class ClassHelperTest {
    @Test
    public void unsafeCastTest() {
        var list = new ArrayList<>();
        List<String> stringList = ClassHelper.uncheckedCast(list);
        Assertions.assertNotNull(stringList);
    }
}

File: helper/src/test/java/com/loudsight/useful/helper/ClassHelperTest.kt
package com.loudsight.useful.helper

import com.loudsight.utilities.helper.ClassHelper
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import java.util.ArrayList

/**
 * Class to hold utilities related to casting of types
 *
 * @author munyengm
 */
class ClassHelperTest {
    @Test
    fun unsafeCastTest() {
        val list: List<*> = ArrayList<Any>()
        val stringList: List<String> = ClassHelper.uncheckedCast(list)
        Assertions.assertNotNull(stringList)
    }
}